src/athena.hpp

```diff
@@ -96,7 +96,7 @@ struct RegionSize { // aggregate and POD type; do NOT reorder member declaration
 
 // needed for arrays dimensioned over grid directions
 // enumerator type only used in Mesh::EnrollUserMeshGenerator()
-enum CoordinateDirection { X1DIR = 0, X2DIR = 1, X3DIR = 2 };
+enum CoordinateDirection { X0DIR = 0, X1DIR = 1, X2DIR = 2, X3DIR = 3 };
```

Comment:
I wanted to do this, but it's annoying.  Wouldn't it require that we always write
```c++
CoordinateDirection::X1DIR
```
for example?

Reply:
Yeah that's true. That would be annoying. You could do:
```C++
enum class CoordinateDirection { X0, X1, X2, X3 };
```
to be a little less verbose, but that would be obnoxious. And you could add a using statement where appropriate:
```C++
using D=CoordinateDirection;
coord.Volume(D::X1,k,j,i);
```
Alternatively,
```C++
namespace  Coordinates {
  enum class Dir { X0=0, X1=1, X2=2, X3=3 };
}
```
so that you could call it like
```C++
using CoordinateDirection::Dir;
coord.Volume(Dir::X0);
```
or something like that. (If I recall, pulling an enum class out of a namespace is slightly more difficult than I wrote above, but not much harder.)